android/app/build.gradle

```diff
@@ -27,8 +27,12 @@ android {
     }
 
     buildTypes {
+        debug {
+            minifyEnabled true
```

Comment:
I'm a bit worried that obfuscating the code may cause problems with crashlytics. Can you check this guide ?
https://docs.fabric.io/android/crashlytics/dex-and-proguard.html